contexts/DonationContext/src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -185,4 +185,9 @@ private function sendDonationConfirmationEmail( Donation $donation ) {
 			$this->mailer->sendConfirmationMailFor( $donation );
 		}
 	}
+
+	private function donationNeedsModeration( AddDonationRequest $donationRequest, Donation $donation ): bool {
+		return !$donation->hasExternalPayment() && $this->policyValidator->needsModeration( $donationRequest );
```

Comment:
You beat me to it! :) But meh... I had not noticed one used the Entity and the other used the request model >_>